mantidproject/mantid
# Fix warnings and errors in docs build

Fixes the warnings & errors that are present in a build of the documentation.

**To test:**

Remove the `build/docs/html` & `build/docs/doctrees` directories and build the documentation. Build the html documentation and check that there are no errors. Note that you will need `MAKE_VATES=ON` to avoid warnings around `SaveWorkspaceToVTK`.

No issue number.

_Does not need to be in the release notes._

---
#### Reviewer

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):
##### Code Review
##### Functional Tests
- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
